src/main/java/appeng/me/GridConnection.java

```diff
@@ -123,11 +122,20 @@ else if( this.isNetworkABetter( a, b ) )
 		}
 
 		// a connection was destroyed RE-PATH!!
-		final IPathingGrid p = this.sideA.getInternalGrid().getCache( IPathingGrid.class );
+		final IPathingGrid p = connection.sideA.getInternalGrid().getCache( IPathingGrid.class );
 		p.repath();
 
-		this.sideA.addConnection( this );
-		this.sideB.addConnection( this );
+		connection.sideA.addConnection( connection );
+		connection.sideB.addConnection( connection );
+
+		return connection;
+	}
+
+	private GridConnection( final GridNode aNode, final GridNode bNode, final AEPartLocation fromAtoB )
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this constructor to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1213)